eos-tech-support/eos-download-image

```diff
@@ -298,7 +298,18 @@ class EosDownloadImage(object):
     def fetch_manifest(self):
         manifest_url = '{base}/releases-{product}-3.json'.format(
             base=self.base, product=self.args.product)
-        manifest = self.session.get(manifest_url).json()
+        response = self.session.get(manifest_url)
+
+        try:
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as e:
+            log("Couldn't fetch manifest for product", self.args.product)
+            log(e)
+            sys.exit(1)
+
+        print(response.headers)
```

Comment:
Ah! You spotted my, uh, check that the reviewers are reviewing. Definitely not testing code left in by accident.